int32 ThumbFinger
int32 IndexFinger
int32 MiddleFinger
int32 RingFinger
int32 LittleFinger
